packages/internal-tooling/babel/index.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
not sure that we should introduce additional package for the tooling stuff - I would rather expect it to be something like `scripts/config/build` dir of the root (private) package. Could you, please, suggest why it is better to have this as package?

Another thing where I see package atomicity being broken is its `lerna-alisases` - essentially, this makes this package to be dependent on the env setup where it is used. You might argue that this is called `internal-tooling`, which makes a hint that it is not universal applicable - and I agree here, but then would still have a question of why we need this as package and not just as a content of some tooling directory in our project :)

Reply:
Agree with your point, `lerna-alias` is a leaky abstraction in this case. It moved mapping to per-package configs, so the main config doesn't  know anything about Lerna now 👍 

_Why we need a separate package?_ I expressed this in the PR description, I want to provide something like Create React App, but for our end-users to allow create and publish their Stardust components.

Reply:
great, let me take a look..

Reply:
have discussed with @layershifter - while it was possible to delay this step of package segregation to the future, there is one very appealing benefit of this move:
- it would allow us to separate `package.json` dependencies on those that are really needed by the logic and those that are necessary for the tooling part (now `internal-tooling` package)